Cannot FInd name Process {N}+Angular2

`import dialogs = require("ui/dialogs");
import {AuthService} from "./auth.service";
import {Router} from '@angular/router';
require( "nativescript-master-technology" );
`

`
....
 if (result){
                      process.exit();
                    }
....`

**Attempting to execute the TS compiler process method returns the following error**

```
13:34:57 - File change detected. Starting incremental compilation...

app/shared/services/application.service.ts(30,52): error TS2304: Cannot find name 'process'.
```

--- a/README.md
+++ b/README.md
@@ -39,6 +39,8 @@ require( "nativescript-master-technology" );
 
  All routines are global; you do not need to keep a reference to the library.
 
+ If you get TypeScript complaining about process not being defined, then put `/// <reference path="./node_modules/nativescript-master-technology/references.d.ts" />` in your `references.d.ts`.
+
 ### Methods
 
 #### setImmediate(<function>);

--- a/references.d.ts
+++ b/references.d.ts
@@ -1,0 +1,20 @@
+declare var global: NodeJS.Global;
+declare var process: Process;
+
+declare namespace NodeJS {
+    interface Global {
+        process: Process;
+    }
+}
+
+interface Process {
+    restart(msg?: string): void;
+    exit(): void;
+    isDebug(): boolean;
+    isEmulator(): boolean;
+    processMessages(): void;
+}
+
+interface Console {
+    keys(data: string | {}, printValue?: boolean): void;
+}

